recipes/glaze/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.51.1"
+
+class GlazeConan(ConanFile):
+    name = "glaze"
+    description = "Extremely fast, in memory, JSON and interface library for modern C++"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/stephenberry/glaze"
+    topics = ("json", "memory", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "11",
+            "clang": "12",
+            "apple-clang": "13.1",
```

Comment:
Use this pattern: https://github.com/conan-io/conan-center-index/blob/583751d39652c88f7cdc6874f3dbbafa961633e8/recipes/spy/all/conanfile.py#L41-L51

Reply:
@SpaceIm 
Thanks!
I try to understand.

Reply:
Without this trick, it raises for `compiler.version=13`, but we want to raise only for `compiler.version=13.0` (or any major version < 13)

Reply:
@SpaceIm 
Now I understand.
Thanks!